packages/core/src/app.ts

```diff
@@ -2086,6 +2106,10 @@ export class App {
      * ================
      */
 
+    private _hasPremiumFeaturesDisabled(features: AccountFeatures | OrgFeatures) {
```

Comment:
I wasn't sure if you wanted to do something different here (or check for more features like `attachments`).